doc/source/building/build_docker_container.rst

```diff
@@ -1,190 +1,91 @@
+* ``labels``: Adds custom metadata to an image, is key-value pair. Equivalent
+  of the `LABEL` directive of a Docker file.
 * ``expose``: Informs at which ports is the container listening at runtime.
-* ``env``: Sets an environment value, is key-value pair.
+  Equivalent of the `EXPOSE` directive of a Docker file.
+* ``environment``: Sets an environment values using key-value pairs.
+  Equivalent of the `env` directive of a Docker file.
 * ``entrypoint``: Sets the executable that the container will run, it can
-  include parameters.
-* ``volume``: Creates a mountpoint with the given name and marks it to hold
-  external volumes from the host or from other contianers.
-* ``user``: Sets the user name or UID to be used when running `cmd` and
-  `entrypoint`.
+  include parameters. Equivalent of the `ENTRYPOINT` directive of a Docker
+  file..
+* ``volumes``: Creates mountpoints with the given name and marks it to hold
+  external volumes from the host or from other contianers. Equivalent of the
+  `VOLUME` directive of a Docker file.
+* ``user``: Sets the user name or UID to be used when running `entrypoint` and
+  `subcommand`. Equivalent of the `USER` directive of a Docker file.
 * ``workdir``: Sets the working directory to be used when running `cmd` and
-  `entrypoint`.
-
-Map DockerFile Directives to KIWI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The native build description in docker is provided by a so called
-DockerFile. The DockerFile is a shell like key=value based file.
-In addition to the metadata information as shown above other
-additional information could be part of a DockerFile. KIWI can
-not natively handle the contents of the DockerFile, thus the
-following list shows how to map the DockerFile directive in
-KIWI:
-
-* `ADD`
+  `entrypoint`. Equivalent of the `WORKDIR` directive of a Docker file.
 
-  In order to include a tarball and auto extract it in KIWI it can be done by
-  including the files in the :ref:`overlay tree <decription_components>`,
-  or by including an `<archive>` item inside the `<packages type="image">`
-  section. **ADD** also handles remote content, in KIWI this should be done
-  within the `config.sh` script.
+Other Docker file directives such as ``RUN`` and ``COPY`` or ``ADD``, can be
+mapped to KIWI by using the :ref:`config.sh <decription_components>`
+script file to run bash commands or the
+:ref:`overlay tree <decription_components>` to include extra files.
 
-* `RUN`
+The following example shows how to build a Docker base image based on
+openSUSE Leap:
 
-  In order to execute commands during build time in KIWI it can be done by
-  extending the `config.sh` script. Note that the `config.sh` is executed
-  after all the packages have been installed and after the overlay tree has
-  been applied.
+1. Make sure you have checked out the example image descriptions,
+   see :ref:`example-descriptions`.
 
-* `COPY`
+2. Install the required toolchain for Docker images building.
```

Comment:
IMHO, I would separate the step of including the repo from installing the packages like this:

```
1. Include the ``Virtualization`` repository to your list:

   .. code:: bash
      $ zypper ar -f http://download.opensuse.org/repositories/Virtualization:/containers/<DIST>

2. Install the :command:`umoci` and :command:`skopeo` tools:
  [...]
```